### ISSUE TITLE
How to install it?

### ISSUE THREAD

**Issue by p6002** [Genymobile/scrcpy#2204]
Dude, why you just don't make exe file, so peoples can use it. You just want to keep this app in secret or what? 

- [ ] I have checked that a similar [feature request](https://github.com/Genymobile/scrcpy/issues?q=is%3Aopen+is%3Aissue+label%3A%22feature+request%22) does not already exist.

**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by rom1v** [comment]
> You just want to keep this app in secret or what?

Yes, exactly. But people keep finding it :disappointed:, I have to better hide it.

> Dude, why you just don't make exe file, so peoples can use it.

At least, I managed to make the README secret.

If you find it and read it, you will find a download link for Windows: https://github.com/Genymobile/scrcpy#get-the-app
The archive contains the .exe and the other necessary files…

**Comment by p6002** [comment]
If these links worked, or if the installation process made sense, then this post wouldn't be here.
Is it really that hard to make an installation file? Your windows installation instructions contain linux commands, then there are some non-cmd commands... How normal people are supposed to know how to use it. It's 2021, let's use single file applications.

Meanwhile, what tf is this? 
https://i.imgur.com/iXx62N7.png

**Comment by RndyP** [comment]
@p6002  Sounds like you're pissed off about a great, free app, that doesn't have a modern Windows installer.... This is old-school install. You first create a folder called srccpy wherever you want it, download the zip file, extract all the files to the folder, and then create a shortcut on your desktop to the scrcpy.exe file in that folder. That's it. Isn't that horrible? If you are having trouble, I can help you with it....

**Comment by csf44033** [comment]
You need Windows 10 to run this program.
lesser versions (8.1) separate devices into 'other', while the correct destination is 'portable'.
